src/pfed.cpp

```diff
@@ -27,7 +27,7 @@ namespace dbm
     void pfed_t::decRef()
     {
         if (!--(ptr->count)) {
-            alloc.destroy(ptr);
+            std::allocator_traits<std::allocator<pfed_t::pfed_s>>::destroy(alloc, ptr);
```

Comment:
this is unreadable.
can we move this code inside `alloc.destroy(ptr)` ?

Reply:
But destroy is a method from the standard library?

Reply:
Fixed it by reverting everything.